kobo/apps/audit_log/models.py

```diff
@@ -14,6 +14,7 @@ class AuditAction(models.TextChoices):
     PUT_BACK = 'put-back', 'PUT BACK'
     REMOVE = 'remove', 'REMOVE'
     UPDATE = 'update', 'UPDATE'
+    AUTH = 'auth', 'AUTH'
```

Comment:
I think this should be added this pr#5035?

Reply:
Good catch. Result of branching shenanigans